actions/src/openstack_actions.py

```diff
@@ -42,4 +43,9 @@ def parse_configs(self, **kwargs):
                 self.config, kwargs["smtp_account_name"]
             )
             del kwargs["smtp_account_name"]
+        if "atlassian_account_name" in kwargs:
+            kwargs["atlassian_account"] = JiraAccount.from_pack_config(
+                self.config, kwargs["atlassian_account_name"]
+            )
+            del kwargs["atlassian_account_name"]
```

Comment:
Why does the `atlassian_account_name` have to be removed from `kwargs` if it is found in kwargs?

Reply:
`atlassian_account_name` is just used to load the correct token and username from the stackstorm config which are then used by the action, `atlassian_account_name` is not needed to connect to jira

Reply:
Could you add a comment for this step in the method?